### ISSUE TITLE
btrfs-sxbackup requires (but does not document) "ionice" on the target

### ISSUE THREAD

**Issue by larsks** [masc3d/btrfs-sxbackup#67]
I tried using `btrfs-sxbackup` to back up to a Synology diskstation device, but the command fails with:

```
ERROR source :: Command ['ssh', '-o', 'ServerAliveInterval=5', '-o', 'ServerAliveCountMax=3', 'backup@btrfs', 'ionice -c3 btrfs receive "/volume1/backups/madhatter/home/"'] failed with error code 127 (bash: ionice: command not found)
```

This was a surprise, because the README only says:

> The following packages have to be available on both source and destination
> 
> - bash
> - btrfs-progs

It would be nice if sxbackup would *not* use ionice by default; if folks want to run `btrfs receive` using ionice, they can use an ssh forced command to modify the `btrfs receive` command line.

**Comment by masc3d** [comment]
yes, README does not include standard packages like `util-linux` or `coreutils` which are provided by most distros.
in the past it has been seen that omitting `ionice` can cause heavy io stalling for other processes during backup.

I don't have synology to test this, as similar other problems could be expected down the road with a heavily modified distro like this. but you could fork your own version and modify it accordingly.

I would accept PR for this, when implemented properly (ionice should remain the default with an option to disable it)
